xc/xc7/tests/soc/litex/base/baselitex_arty.xdc

```diff
@@ -271,7 +271,7 @@ set_property INTERNAL_VREF 0.750 [get_iobanks 34]
 
 #create_clock -name sys_clk -period 16.666 [get_nets sys_clk]
 #
-#create_clock -name clk100 -period 10.0 [get_nets clk100]
+create_clock -name clk100 -period 10.0 [get_ports clk100]
```

Comment:
`get_nets` will basically return the same thing as `get_ports` in this case, but if it is us creating these XDCs we can just skip the `get_` commands and make `clk100` alone.

Reply:
set to `clk100` alone